components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -81,6 +85,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
 
     // Set up ink controller to splash ink on taps.
     _inkTouchController = [[MDCInkTouchController alloc] initWithView:self];
+    _inkTouchController.delegate = self;
     [_inkTouchController addInkView];
```

Comment:
Did we consider:
[self sendSubviewToBack:_inkTouchController.defaultInkView];

This may be a bit simpler than adding a delegate and implementing a method.